### ISSUE TITLE
ArgumentOutOfRangeException when multi-line array entry is missing final parenthesis and user hits ENTER twice

### ISSUE THREAD

**Issue by ssg47** [PowerShell/PSReadLine#1764]
I tried to paste in an array from a script but didn't notice that the final close-parenthesis was missing.  
I think I hit Ctrl + C to try to terminate the entry, like I would in a normal Windows PowerShell window.  Probably several times.  And then the Up Arrow so I could try again.  But I'm not really sure.
I'm not sure if I didn't notice the error initially (hence the multiple instances in the output, below) because the window wasn't large enough to display the error and the array.
(See end for version info.)
```
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties = @(
>>             'comment',                            # checked
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'division',
>>             'employeeid',
>>             'givenName',
>>             'initials',
>>             'customatt-csUTOControlFlags',
>>             'l',
>>             'manager',
>>             'memberOf',
>>             'msDS-cloudExtensionAttribute11',
>>             'msDS-cloudExtensionAttribute12',
>>             'msDS-cloudExtensionAttribute13',
>>             'msDS-cloudExtensionAttribute14',
>>             'name',
>>             'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
>>             'physicalDeliveryOfficeName',
>>             'sAMAccountName',
>>             'scriptPath',
>>             'sn',
>>             'street',
>>             'title',
>>             'userAccountControl',
>>             'userPrincipalName',
>>             'Enabled'
>>
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 Space Space Space Space Space ' s A M A c c o u n t N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s c r i p t P a t h ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s n ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s t r e e t ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' t i t l e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r A c c o u n t C o n t r o l ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r P r i n c i p a l N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' E n a b l e d ' Enter
 Enter


Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.AcceptLineImpl(Boolean validate)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties = @(
>>             'comment',                            # checked
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'division',
>>             'employeeid',
>>             'givenName',
>>             'initials',
>>             'customatt-csUTOControlFlags',
>>             'l',
>>             'manager',
>>             'memberOf',
>>             'msDS-cloudExtensionAttribute11',
>>             'msDS-cloudExtensionAttribute12',
>>             'msDS-cloudExtensionAttribute13',
>>             'msDS-cloudExtensionAttribute14',
>>             'name',
>>             'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
>>             'physicalDeliveryOfficeName',
>>             'sAMAccountName',
>>             'scriptPath',
>>             'sn',
>>             'street',
>>             'title',
>>             'userAccountControl',
>>             'userPrincipalName',
>>             'Enabled'
>>
>>
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 Space Space Space Space ' s A M A c c o u n t N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s c r i p t P a t h ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s n ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s t r e e t ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' t i t l e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r A c c o u n t C o n t r o l ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r P r i n c i p a l N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' E n a b l e d ' Enter
 Enter
 Enter


Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -2.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.AcceptLineImpl(Boolean validate)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties = @(
>>             'comment',                            # checked
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'division',
>>             'employeeid',
>>             'givenName',
>>             'initials',
>>             'customatt-csUTOControlFlags',
>>             'l',
>>             'manager',
>>             'memberOf',
>>             'msDS-cloudExtensionAttribute11',
>>             'msDS-cloudExtensionAttribute12',
>>             'msDS-cloudExtensionAttribute13',
>>             'msDS-cloudExtensionAttribute14',
>>             'name',
>>             'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
>>             'physicalDeliveryOfficeName',
>>             'sAMAccountName',
>>             'scriptPath',
>>             'sn',
>>             'street',
>>             'title',
>>             'userAccountControl',
>>             'userPrincipalName',
>>             'Enabled'
>>
>>
>>
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 Space Space Space ' s A M A c c o u n t N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s c r i p t P a t h ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s n ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s t r e e t ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' t i t l e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r A c c o u n t C o n t r o l ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r P r i n c i p a l N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' E n a b l e d ' Enter
 Enter
 Enter
 Enter


Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -3.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.AcceptLineImpl(Boolean validate)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties = @(
>>             'comment',                            # checked
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'division',
>>             'employeeid',
>>             'givenName',
>>             'initials',
>>             'customatt-csUTOControlFlags',
>>             'l',
>>             'manager',
>>             'memberOf',
>>             'msDS-cloudExtensionAttribute11',
>>             'msDS-cloudExtensionAttribute12',
>>             'msDS-cloudExtensionAttribute13',
>>             'msDS-cloudExtensionAttribute14',
>>             'name',
>>             'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
>>             'physicalDeliveryOfficeName',
>>             'sAMAccountName',
>>             'scriptPath',
>>             'sn',
>>             'street',
>>             'title',
>>             'userAccountControl',
>>             'userPrincipalName',
>>             'Enabled'
>>
>>
>>
>>
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 Space Space ' s A M A c c o u n t N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s c r i p t P a t h ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s n ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s t r e e t ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' t i t l e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r A c c o u n t C o n t r o l ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r P r i n c i p a l N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' E n a b l e d ' Enter
 Enter
 Enter
 Enter
 Enter


Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -4.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.AcceptLineImpl(Boolean validate)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties = @(
>>             'comment',                            # checked
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'division',
>>             'employeeid',
>>             'givenName',
>>             'initials',
>>             'customatt-csUTOControlFlags',
>>             'l',
>>             'manager',
>>             'memberOf',
>>             'msDS-cloudExtensionAttribute11',
>>             'msDS-cloudExtensionAttribute12',
>>             'msDS-cloudExtensionAttribute13',
>>             'msDS-cloudExtensionAttribute14',
>>             'name',
>>             'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
>>             'physicalDeliveryOfficeName',
>>             'sAMAccountName',
>>             'scriptPath',
>>             'sn',
>>             'street',
>>             'title',
>>             'userAccountControl',
>>             'userPrincipalName',
>>             'Enabled'
>>
>>
>>
>>
>>
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 Space ' s A M A c c o u n t N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s c r i p t P a t h ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s n ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s t r e e t ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' t i t l e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r A c c o u n t C o n t r o l ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r P r i n c i p a l N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' E n a b l e d ' Enter
 Enter
 Enter
 Enter
 Enter
 Enter


Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -5.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.AcceptLineImpl(Boolean validate)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties = @(
>>             'comment',                            # checked
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'division',
>>             'employeeid',
>>             'givenName',
>>             'initials',
>>             'customatt-csUTOControlFlags',
>>             'l',
>>             'manager',
>>             'memberOf',
>>             'msDS-cloudExtensionAttribute11',
>>             'msDS-cloudExtensionAttribute12',
>>             'msDS-cloudExtensionAttribute13',
>>             'msDS-cloudExtensionAttribute14',
>>             'name',
>>             'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
>>             'physicalDeliveryOfficeName',
>>             'sAMAccountName',
>>             'scriptPath',
>>             'sn',
>>             'street',
>>             'title',
>>             'userAccountControl',
>>             'userPrincipalName',
>>             'Enabled'
>>
>>
>>
>>
>>
>>
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 ' s A M A c c o u n t N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s c r i p t P a t h ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s n ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s t r e e t ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' t i t l e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r A c c o u n t C o n t r o l ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r P r i n c i p a l N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' E n a b l e d ' Enter
 Enter
 Enter
 Enter
 Enter
 Enter
 Enter


Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -6.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.AcceptLineImpl(Boolean validate)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties = @(
>>             'comment',                            # checked
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'division',
>>             'employeeid',
>>             'givenName',
>>             'initials',
>>             'customatt-csUTOControlFlags',
>>             'l',
>>             'manager',
>>             'memberOf',
>>             'msDS-cloudExtensionAttribute11',
>>             'msDS-cloudExtensionAttribute12',
>>             'msDS-cloudExtensionAttribute13',
>>             'msDS-cloudExtensionAttribute14',
>>             'name',
>>             'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
>>             'physicalDeliveryOfficeName',
>>             'sAMAccountName',
>>             'scriptPath',
>>             'sn',
>>             'street',
>>             'title',
>>             'userAccountControl',
>>             'userPrincipalName',
>>             'Enabled'
>>
>>
>>
>>
>>
>>
>>
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 A M A c c o u n t N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s c r i p t P a t h ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s n ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s t r e e t ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' t i t l e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r A c c o u n t C o n t r o l ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r P r i n c i p a l N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' E n a b l e d ' Enter
 Enter
 Enter
 Enter
 Enter
 Enter
 Enter
 Ctrl+x )

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -7.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties = @(
>>             'comment',                            # checked
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'division',
>>             'employeeid',
>>             'givenName',
>>             'initials',
>>             'customatt-csUTOControlFlags',
>>             'l',
>>             'manager',
>>             'memberOf',
>>             'msDS-cloudExtensionAttribute11',
>>             'msDS-cloudExtensionAttribute12',
>>             'msDS-cloudExtensionAttribute13',
>>             'msDS-cloudExtensionAttribute14',
>>             'name',
>>             'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
>>             'physicalDeliveryOfficeName',
>>             'sAMAccountName',
>>             'scriptPath',
>>             'sn',
>>             'street',
>>             'title',
>>             'userAccountControl',
>>             'userPrincipalName',
>>             'Enabled')
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties
comment
company
department
description
displayName
distinguishedName
division
employeeid
givenName
initials
customatt-csUTOControlFlags
l
manager
memberOf
msDS-cloudExtensionAttribute11
msDS-cloudExtensionAttribute12
msDS-cloudExtensionAttribute13
msDS-cloudExtensionAttribute14
name
personalTitle
physicalDeliveryOfficeName
sAMAccountName
scriptPath
sn
street
title
userAccountControl
userPrincipalName
Enabled
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties = @(
>>             'comment',                            # checked
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'division',
>>             'employeeid',
>>             'givenName',
>>             'initials',
>>             'customatt-csUTOControlFlags',
>>             'l',
>>             'manager',
>>             'memberOf',
>>             'msDS-cloudExtensionAttribute11',
>>             'msDS-cloudExtensionAttribute12',
>>             'msDS-cloudExtensionAttribute13',
>>             'msDS-cloudExtensionAttribute14',
>>             'name',
>>             'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
>>             'physicalDeliveryOfficeName',
>>             'sAMAccountName',
>>             'scriptPath',
>>             'sn',
>>             'street',
>>             'title',
>>             'userAccountControl',
>>             'userPrincipalName',
>>             'Enabled'
>>
>>
>>
>>
>>
>>
>> )
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 u n t N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s c r i p t P a t h ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s n ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s t r e e t ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' t i t l e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r A c c o u n t C o n t r o l ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r P r i n c i p a l N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' E n a b l e d ' ) Enter
 $ p r o Tab Tab Tab Enter
 UpArrow UpArrow UpArrow UpArrow UpArrow

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -7.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.HistoryRecall(Int32 direction)
   at Microsoft.PowerShell.PSConsoleReadLine.PreviousHistory(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties = @(
>>             'comment',                            # checked
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'division',
>>             'employeeid',
>>             'givenName',
>>             'initials',
>>             'customatt-csUTOControlFlags',
>>             'l',
>>             'manager',
>>             'memberOf',
>>             'msDS-cloudExtensionAttribute11',
>>             'msDS-cloudExtensionAttribute12',
>>             'msDS-cloudExtensionAttribute13',
>>             'msDS-cloudExtensionAttribute14',
>>             'name',
>>             'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
>>             'physicalDeliveryOfficeName',
>>             'sAMAccountName',
>>             'scriptPath',
>>             'sn',
>>             'street',
>>             'title',
>>             'userAccountControl',
>>             'userPrincipalName',
>>             'Enabled'
>>
>>
>>
>>
>>
>>
>> )
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 Space Space Space Space ' s c r i p t P a t h ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s n ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s t r e e t ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' t i t l e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r A c c o u n t C o n t r o l ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r P r i n c i p a l N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' E n a b l e d ' ) Enter
 $ p r o Tab Tab Tab Enter
 UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow DownArrow DownArrow DownArrow UpArrow UpArrow UpArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow UpArrow UpArrow UpArrow UpArrow UpArrow

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -7.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.HistoryRecall(Int32 direction)
   at Microsoft.PowerShell.PSConsoleReadLine.PreviousHistory(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\MyUserId\Documents\WindowsPowerShell> $properties = @(
>>             'comment',                            # checked
>>             'company',
>>             'department',
>>             'description',
>>             'displayName',
>>             'distinguishedName',
>>             'division',
>>             'employeeid',
>>             'givenName',
>>             'initials',
>>             'customatt-csUTOControlFlags',
>>             'l',
>>             'manager',
>>             'memberOf',
>>             'msDS-cloudExtensionAttribute11',
>>             'msDS-cloudExtensionAttribute12',
>>             'msDS-cloudExtensionAttribute13',
>>             'msDS-cloudExtensionAttribute14',
>>             'name',
>>             'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
>>             'physicalDeliveryOfficeName',
>>             'sAMAccountName',
>>             'scriptPath',
>>             'sn',
>>             'street',
>>             'title',
>>             'userAccountControl',
>>             'userPrincipalName',
>>             'Enabled'
>>
>>
>>
>>
>>
>>
>> )
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s n ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' s t r e e t ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' t i t l e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r A c c o u n t C o n t r o l ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' u s e r P r i n c i p a l N a m e ' , Enter
 Space Space Space Space Space Space Space Space Space Space Space Space ' E n a b l e d ' ) Enter
 $ p r o Tab Tab Tab Enter
 UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow DownArrow DownArrow DownArrow UpArrow UpArrow UpArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow DownArrow UpArrow UpArrow UpArrow UpArrow UpArrow

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -7.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.HistoryRecall(Int32 direction)
   at Microsoft.PowerShell.PSConsoleReadLine.PreviousHistory(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\MyUserId\Documents\WindowsPowerShell>  
```

This is in 
Windows PowerShell 
```
$psversiontable

Name                           Value
----                           -----
PSVersion                      5.1.18362.752
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.18362.752
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```
in
```
Windows Terminal
Version: 0.11.1121.0
```

**Comment by ssg47** [comment]
To recreate:

In Windows Terminal, start a new instance of Windows PowerShell.
Paste in the following:
```
$properties = @(
            'comment',                            # checked
            'company',
            'department',
            'description',
            'displayName',
            'distinguishedName',
            'division',
            'employeeid',
            'givenName',
            'initials',
            'customatt-csUTOControlFlags',
            'l',
            'manager',
            'memberOf',
            'msDS-cloudExtensionAttribute11',
            'msDS-cloudExtensionAttribute12',
            'msDS-cloudExtensionAttribute13',
            'msDS-cloudExtensionAttribute14',
            'name',
            'personalTitle',  # xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
            'physicalDeliveryOfficeName',
            'sAMAccountName',
            'scriptPath',
            'sn',
            'street',
            'title',
            'userAccountControl',
            'userPrincipalName',
            'Enabled'
```
and hit ENTER twice.

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.4 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.
